test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
```

Comment:
Is this relevant?
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go#L107
The top of the function calls NewDefaultFramework().